common/server_player.cpp

```diff
@@ -485,8 +515,18 @@ Response::ResponseCode Server_Player::moveCard(GameEventStorage &ges,
             event.set_card_id(static_cast<google::protobuf::uint32>(card->getId()));
             ges.enqueueGameEvent(event, playerId);
 
-            card->deleteLater();
-        } else {
+            if (Server_Card *stashedCard = card->takeStashedCard()) {
+                stashedCard->setId(newCardId());
+                ges.enqueueGameEvent(makeCreateTokenEvent(startzone, stashedCard, card->getX(), card->getY()),
+                                     playerId);
+                card = stashedCard;
```

Comment:
I don't see card being deleted before this, this will probably leak memory

Reply:
`card->deleteLater()` did get incorrectly yanked from this branch.